podpac/core/coordinates/coordinates.py

```diff
@@ -784,7 +802,7 @@ def udrop(self, dims, ignore_missing=False):
                 if c.name not in dims:
                     cs.append(c)
             elif isinstance(c, StackedCoordinates):
-                stacked = [s for s in c if s.name not in dims]
+                stacked = [s for s in c.values() if s.name not in dims]

```

Comment:
Let's try to use either `c.values()` or `c._coords.values()` throughout; I think we have both at the moment.